### ISSUE TITLE
Builds are not triggered on main branch so container image isn't produced

### ISSUE THREAD

**Issue by jsturtevant** [prometheus-community/windows_exporter#940]
In https://github.com/prometheus-community/windows_exporter/pull/864 some logic was added to build a "bleeding edge" container:

https://github.com/prometheus-community/windows_exporter/blob/5106b829c3092ef3f1346790336ba0789ef30c01/.github/workflows/ci.yml#L180-L185

This doesn't trigger currently because the workflow only runs on PR's and tags:

https://github.com/prometheus-community/windows_exporter/blob/5106b829c3092ef3f1346790336ba0789ef30c01/.github/workflows/ci.yml#L5-L12

Adding a trigger like 

```
on:
  push:
    branches: [ master ]
``` 

would build, test and then push the bleeding edge image as well give a signal that main branch is healthy after the PR merge. @breed808 @carlpett  any thoughts on this?

**Comment by breed808** [comment]
Agreed, we'll need to enable CI for pushes to `master` to get this working.

**Comment by breed808** [comment]
`master` CI job is failing when attempting to build container images. Will need to investigate and resolve.

```
docker build --build-arg=BASE=mcr.microsoft.com/windows/nanoserver:1809 -f Dockerfile -t ghcr.io/prometheus-community/windows-exporter:0.18.2+11-1809 .
invalid argument "ghcr.io/prometheus-community/windows-exporter:0.18.2+11-1809" for "-t, --tag" flag: invalid reference format
See 'docker build --help'.
```

**Comment by breed808** [comment]
I think the plus in the tag name is causing the failure. From the [docker documentation](https://docs.docker.com/engine/reference/commandline/tag/):

> A tag name must be valid ASCII and may contain lowercase and uppercase letters, digits, underscores, periods and dashes. A tag name may not start with a period or a dash and may contain a maximum of 128 characters.

**Comment by jsturtevant** [comment]
Thanks for working through the fixes!  Sorry these slipped through my testing.  

Looks like the image is available now: https://github.com/prometheus-community/windows_exporter/pkgs/container/windows-exporter 🥳
